pkg/trace/obfuscate/json.go

```diff
@@ -25,27 +27,47 @@ func (o *Obfuscator) obfuscateJSON(span *pb.Span, tag string, obfuscator *jsonOb
 	// as much of it as we could. It is safe to accept the output, even if partial.
 }
 
+type valueTransformer func(string) string
```

Comment:
Is there any value on having this type? It's pretty straightforward. You can keep it if you like, but I'm thinking just naming things `var transformerFunc func(string) string` will be equally understandable, easy to type and read.

Reply:
Good point, removed